### DISCUSSION TITLE
Closed Captions / Subtitle tracks

### DISCUSSION THREAD

**Discussion by tlsharkey** [aclap-dev/video-downloadhelper#298] [Help and Q&A]
### Your question

Trying to find a way to get captions from videos. Any suggestions / planned future support?

Thanks, great tool.

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/tlsharkey",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by GrampaWildWilly** [comment]
At the moment, not with VDH.  But we have been promised there will be some work to be done on this in the medium term.  Meantime, you can use ffmpeg to get captions.  I have some examples of that in this forum & in our previous forum on Google Groups.
